packages/client/scripts/default-index.d.ts

```diff
@@ -33,16 +33,25 @@ export declare const PrismaClient: any
  */
 export declare type PrismaClient = any
 
-export declare type PrismaClientExtends<ExtArgs extends runtime.Types.Extensions.Args = runtime.Types.Extensions.DefaultArgs> = {
+export declare type PrismaClientExtends<
```

Comment:
Fommating is making this hard to view, the change is actually one line: 

```diff
- => PrismaClientExtends<runtime.Types.Utils.PatchDeep<Args, ExtArgs>>)
+ => PrismaClientExtends<runtime.Types.Utils.PatchDeep<Args, PrismaClientExtends<ExtArgs>>)
```

Reply:
I think this is incorrect. I have simplified this in another branch to be replaced with a simple intersection.